data/campaigns/Under_the_Burning_Suns/scenarios/05_A_Subterranean_Struggle.cfg

```diff
@@ -673,16 +673,19 @@
                 [/message]
             [/then]
         [/if]
-
-        [move_unit_fake]
+        [unit]
             type=Dwarvish Fighter
             side=4
-            x=25,24,23,22,21
-            y=28,28,29,28,29
-        [/move_unit_fake]
-
-        #TODO maybe the wounded dwarf should really be wounded?
-        {NAMED_NOTRAIT_UNIT 4 (Dwarvish Fighter) 21 29 (Wounded Dwarf) ( _ "Wounded Dwarf")}
```

Comment:
What is the name of the dwarf now - does he get a random one?
(I'm not opposed to that)

Reply:
Name is currently simply "Wounded Dwarf".

Reply:
I think that part is not in the code anymore - you removed it here, but it's nowhere else, only as ID.
Again, I consider this not a bad idea by default because UtBS has often long names which become too long for the sidebar if translated in other languages. But I didn't played this scenario yet. 
(That's why I would prefer if someone else merges this, though looks like good to go)

Reply:
Apologies, yes you're right this now gives a random dwarf name. That was an unintentional oversight, (just when you think you've done enough testing!) though actually I think I prefer it this way as he's now evidently wounded.